docs/guides/v7-deprecated-props-and-components.md

```diff
@@ -54,80 +57,81 @@ For more information, click the name of the component to see its full documentat
+| [TextInput](#TextInput)        | label                    | renderLabel                   | yes     |                                                                                                                                                             |
+|                                | required                 | isRequired                    | yes     |                                                                                                                                                             |
+|                                | inline                   | display                       | yes     | use `display` ('inline-block' or 'block') instead                                                                                                           |
+|                                | icon                     | renderAfterInput              | yes     |                                                                                                                                                             |
+| [TimeSelect](#TimeSelect)      | label                    | renderLabel                   | yes     |                                                                                                                                                             |
+| [Tooltip](#Tooltip)            | tip                      | renderTip                     | yes     | use `renderTip` instead (`renderTip` property is now required)                                                                                              |
+|                                | variant                  | color                         | yes     | use `color` ('primary' or 'primary-inverse') instead                                                                                                        |
+| [View](#View)                  | focused                  | withFocusOutline              | yes     |                                                                                                                                                             |
+|                                | visualDebug              | withVisualDebug               | yes     |                                                                                                                                                             |
 
 ### Deprecated Property Values
 
 The following table shows component properties that have changes in the set of values they accept.
 
-The listed `delimiter` values were deleted from `List` and `List.Item`, so they have to be changed manually, but the rest of the changes are covered with [codemods](#ui-codemods).
+The listed `delimiter` values were deleted from `List` and `List.Item`, so they have to be changed manually (the `updateV7Props` codemod will output a warning where you need to do so), but the rest of the changes are covered with [codemods](#ui-codemods).
```

Comment:
To be honest it is not 100% clear when to use what codemods now. There is this `updateV7Props`, that should be used from v8, and there are still links to the [ui-codemods](https://61fce848dc1a4c605ce5cde1--preview-instui.netlify.app/#ui-codemods) page but that lists different codemods and talks about the cli... Could you please add more info to the ui-codemods page too to make things clear?

Reply:
added a big warning to the v7 readme :)